common/src/main/java/com/kumuluz/ee/configuration/sources/FileConfigurationSource.java

```diff
@@ -48,6 +48,17 @@ public FileConfigurationSource() {
         this.ymlFileName = "config.yml";
         this.yamlFileName = "config.yaml";
         this.propertiesFileName = "config.properties";
+
+        String configurationFileName = System.getProperty("com.kumuluz.ee.configuration.file");
+        if (configurationFileName != null && !configurationFileName.isEmpty()) {
+            if (".yml".equals(configurationFileName.substring(configurationFileName.length() - 4))) {
+                this.ymlFileName = configurationFileName;
+            } else if (".yaml".equals(configurationFileName.substring(configurationFileName.length() - 5))) {
+                this.yamlFileName = configurationFileName;
+            } else if (".properties".equals(configurationFileName.substring(configurationFileName.length() - 11))) {
+                this.propertiesFileName = configurationFileName;
+            }
```

Comment:
Maybe if the file type can not be inferred by it's extension name, we can try and read it as a property file by default (or maybe .yml?) and let it fail if it's not in the correct format, so that it's not simply ignored.

Reply:
Fixed. If the file type can not be inferred by it's extension name, try to load it is as a YML file.

Reply:
Another fix: if configuration file name is provided, framework first tries to load is as YAML file, then as a .properties file